docs/core/introduction.md

```diff
@@ -6,404 +6,52 @@ ms.date: 02/24/2022
 ms.custom: "updateeachrelease"
 recommendations: false
 ---
-# What is .NET? Introduction and overview
+# What is .NET?
```

Comment:
https://www.google.com/search?q=What+is+.NET will point you to https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet

How do we think about this doc vs. the content that the search engines actually point you to?

Reply:
Should we be writing this topic differently, then?

Reply:
By limiting the range of content in the article, the PR as it stands seems to aim at the dotnet.microsoft.com goal of helping people "decide if they want to learn the platform" whereas the more comprehensive introduction offered by the article as it stands gets them started "learning the platform," by introducing more topics and linking to documentation that goes into more depth on those concepts.